express.js

```diff
@@ -1,9 +1,76 @@
+const express = require('express')
+const path = require('node:path')
+const { PORT, ROUTES, REQUEST_METHODS } = require('./contants.js')
+
+const app = express()
+
+//* secio-note (1)
+app.disable('x-powered-by')
+
+//* secio-note (2)
+app.use(express.json())
+
+//* secio-note (3)
+// @midudev ¿Si hubiesen archivos que no son estáticos u otros ficheros dentro del path especificado también son accesibles a la hora de entrar a sus respectivos paths?
+app.use(express.static(path.join(__dirname, 'assets')))
+
+app.all(ROUTES.ROOT, (req, res) => {
+  if (req.method === REQUEST_METHODS.GET) {
+    return res.status(200).send('<h1>¡Hola mundo!</h1>')
+  }
+  return res.status(405).end()
+})
+app.all(ROUTES.LOGO, (req, res) => {
+  //* secio-note (4)
+  if (req.method === REQUEST_METHODS.GET) {
+    const logoPath = path.join('assets', 'logo.webp')
+    // @midudev es la primera vez que veo "sendFile" por lo que no se si sea necesario pasarle ese objeto con root o si simplemente es mejor enviar el path completo.
```

Comment:
En este caso ya estás usando el middleware `express.static` y esto que haces aquí no sería necesario.

El `sendFile` sería para hacer manualmente lo que de forma automática ya te hace el middleware.

Por lo demás, has hecho bien pasando el root.

Reply:
Oh entiendo, muchas gracias por explicarme ello midu, entonces en caso quiera enviar solo un archivo que no este dentro de lo especificado en el middleware express.static, con un sendFile seria lo indicado ? 
```suggestion
    // @midudev es la primera vez que veo "sendFile" por lo que no se si sea necesario pasarle ese objeto con root o si simplemente es mejor enviar el path completo.
```